_data/team.yml

```diff
@@ -239,3 +239,13 @@
   twitter: panavtec
   github: panavtec
 
+- id: charlie
+  name: Carlos Esteban Raffellini
+  image: /assets/img/custom/team/carlos_raffellini.jpg
+  role: Apprentice
+  fullDescription: >
+    <p>Carlos has worked in the e-commerece industry for over 4 year. He enjoys sharing and improving his software development knowledge.</p>
```

Comment:
`Carlos has worked in the e-commerce industry for over four years. He enjoys sharing and improving his software development knowledge.`